### ISSUE TITLE
Prep sales forecaster

### ISSUE THREAD

**Issue by leaflettuce** [leaflettuce/tdwpDB#7]
**Prep train/test sets**
-Split out validation set of NSNM.
-Test set from week one WRAABB

**Prep Vars**
-Standardize numeric
-Dummy out categorical

**Model**
-Attempt some regression
-Regularization

**Implement**
-Trial run: Week 2 & 3 WRAABB.

**Comment by leaflettuce** [comment]
Setup complete for sales only model. 

next steps
-> explore and test models for sales
-> create pipeline for new predictions

**Comment by leaflettuce** [comment]
-r^2 of .42
- tends to overpredict WRA ( avg ~500)
